uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1788,13 +1813,13 @@ sub getUploadIDUsingTarchiveSrcLoc {
 
 =head3 spool($message, $error, $upload_id, $verb)
 
-Calls the Notify->spool function to log all messages.
+Calls the C<Notify->spool> function to log all messages.
 
 INPUTS:
   - $message   : message to be logged in the database
-  - $error     : if 'Y' it's an error log,
+  - $error     : 'Y' for an error log,
                  'N' otherwise
-  - $upload_id : the upload_id
+  - $upload_id : the ID of the upload
```

Comment:
I would say  upload ID (as this is how you seemed to have done it throughout)

Reply:
done